core/controllers/acl_decorators.py

```diff
@@ -2433,3 +2433,50 @@ def test_can_accept_suggestion(
         return test_can_accept_suggestion
 
     return generate_decorator_for_handler
+
+
+def can_submit_answer_details(handler):
+    """Decorator to check whether user can submit answer details.
+
+    Args:
+        handler: function. The function to be decorated.
+
+    Returns:
+        function. The newly decorated function that now can check
+            if users can submit answer details.
+    """
+    def test_can_submit(self, entity_type, entity_id, **kwargs):
+        """Checks if the user can submit answer details.
+
+        Args:
+            entity_type: str. The type of entity.
+            entity_id: str. The ID of the entity.
+            **kwargs: *. Keyword arguments.
+
+        Returns:
+            *. The return value of the decorated function.
+
+        Raises:
+            PageNotFoundException: The page is not found.
+        """
+        if entity_type == feconf.ENTITY_TYPE_EXPLORATION:
+            if entity_id in feconf.DISABLED_EXPLORATION_IDS:
```

Comment:
If it's entirely similar to can_play_exploration, can you find a way to use that decorator here? (avoid duplicate code.)

Reply:
Okay @DubeySandeep I will find a way.